service/src/test/java/org/mskcc/smile/service/SampleServiceTest.java

```diff
@@ -323,4 +323,33 @@ public void testFindSamplesByDate() throws Exception {
                 sampleRepository.findLatestSampleMetadataBySmileId(updatedSample.getSmileSampleId());
         Assertions.assertThat(latestMetadata.getCmoSampleName()).isEqualTo("C-LATESTLABEL-T02-d002");
     }
+
+    /**
+     * Tests if samples found by a valid uuid and igoId are the same(not null)
+     * @throws Exception
+     */
+    @Test
+    public void testFindSampleByUuid() throws Exception {
+        String igoId = "MOCKREQUEST1_B_2";
+
+        SmileSample sample = sampleService.getSampleByInputId(igoId);
+        String uuid = sample.getSmileSampleId().toString();
```

Comment:
instead of uuid let's do investigator id or cmo sample name since the uuid here is set from the instance of `SmileSample` that was fetched by the igoId